roles/openshift_control_plane/tasks/main.yml

```diff
@@ -82,6 +82,17 @@
   - item.kind == 'OpenIDIdentityProvider'
   with_items: "{{ openshift_master_identity_providers }}"
 
+- name: Create the GitHub (Enterprise) ca file if needed
+  copy:
+    dest: "/etc/origin/master/{{ item.name }}_github_ca.crt"
+    content: "{{ openshift.master.github_ca }}"
+    mode: 0600
+    backup: yes
+  when:
+  - openshift.master.github_ca is defined
+  - item.kind == 'GitHubIdentityProvider'
```

Comment:
`roles/lib_utils/filter_plugins/openshift_master.py` needs to be updated to have this class

Reply:
Any ref to this?

Reply:
Oh, nvm, its already there

Reply:
Not entirely sure how Github Enterprise works though - I guess it needs a custom URL too?

Reply:
hostname according to the https://docs.okd.io/3.11/install_config/configuring_authentication.html#GitHub documentation